wrapper.ts

```diff
@@ -100,6 +103,9 @@ export class Pty {
     });
 
     write.on('error', (err: NodeJS.ErrnoException) => {
+      if (err.code && err.code.indexOf('EINTR') !== -1) {
```

Comment:
this seems sufficient to stop the EINTR

```suggestion
      if (err.code && err.code === 'EINTR') {
```

^ is this not safe?